book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] I believe that you are missing a method called `add_rental` in this file, as this is necessary to create the association between book and rental because if you rent a book, you need to make a new rental for it. Here's a slight hint on what it should look like:
```
def add_rental(person, date)
    Rental.new( params here )
end
```
So please make sure to work on this issue.